src/passwordValidation.js

```diff
@@ -21,6 +19,14 @@ export default function isValidPassword(password = "") {
   let majiscule = /^[A-Z0-9]+$/;
   let miniscule = /^[a-z0-9]+$/;
```

Comment:
I would suggest avoiding the use of variable names that are difficult to understand. However, since I can't see where you used these variables, I recommend removing them from here.